Pizzeria.Store/Program.cs

```diff
@@ -13,13 +13,15 @@
 
 var builder = WebApplication.CreateBuilder(args);
 
-var postgresSqlConnectionString = builder.Configuration.GetConnectionString("Postgres");
+builder.AddServiceDefaults();
+
+var postgresSqlConnectionString = builder.Configuration.GetConnectionString(ServiceNames.DatabaseServer);
```

Comment:
This returns null

Reply:
I think it should be `builder.Configuration.GetConnectionString(ServiceNames.StoreDatabase )`